apps/jupyterlab/app/clip.ipynb

```diff
@@ -0,0 +1,110 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# CLIP Embeddings and NLP prompt\n",
+    "\n",
+    "If you run the [Generate Embeddings Workflow](https://docs.aperturedata.io/workflows/embeddings_extraction), \n",
```

Comment:
Can we also remind user to also have some images before having embedding workflow. Also "a photo of a group of more than 2 people" works for certain images, in the comments, let's have some more explanations of good search terms.

Reply:
done